.ci/oci-e2e-deployment.sh

```diff
@@ -9,39 +9,56 @@ export WORKSPACE=$(dirname $(dirname $(readlink -f "$0")));
 export APPLICATION_NAMESPACE="openshift-gitops"
 export APPLICATION_NAME="all-components-staging"
 
+export TEST_BRANCH_ID=$(date +%s)
+export MY_GIT_FORK_REMOTE="qe"
+export MY_GITHUB_ORG="redhat-appstudio-qe"
+export MY_GITHUB_TOKEN="${GITHUB_TOKEN}"
+
 # Available openshift ci environments https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables
 export ARTIFACTS_DIR=${ARTIFACT_DIR:-"/tmp/appstudio"}
 
+command -v yq >/dev/null 2>&1 || { echo "yq is not installed. Aborting."; exit 1; }
+command -v kubectl >/dev/null 2>&1 || { echo "kubectl is not installed. Aborting."; exit 1; }
+command -v e2e-appstudio >/dev/null 2>&1 || { echo "e2e-appstudio bin is not installed. Please install it from: https://github.com/redhat-appstudio/e2e-tests."; exit 1; }
+
+if [[ -z "${GITHUB_TOKEN}" ]]; then
+  echo - e "[ERROR] GITHUB_TOKEN env is not set. Aborting."
+fi
+
+if [[ -z "${QUAY_TOKEN}" ]]; then
+  echo - e "[ERROR] QUAY_TOKEN env is not set. Aborting."
+fi
+
 #Stop execution on any error
 trap "catchFinish" EXIT SIGINT
 
+# Don't remove appstudio. Can broke development cluster
 function catchFinish() {
     JOB_EXIT_CODE=$?
     if [[ "$JOB_EXIT_CODE" != "0" ]]; then
         echo "[ERROR] Job failed with code ${JOB_EXIT_CODE}."
     else
         echo "[INFO] Job completed successfully."
     fi
-    /bin/bash "$WORKSPACE"/hack/destroy-cluster.sh
 
+    git remote rm $MY_GIT_FORK_REMOTE
```

Comment:
this is just removing local branch, which is in the test environment so it's probably not needed. Maybe you would rather remove pushed branch: `git push qe --delete review-main-${TEST_BRANCH_ID}`